#!/usr/bin/env python
PACKAGE = "myRace"
import roslib
roslib.load_manifest(PACKAGE)
from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("threshold", int_t, 0, "threshold of image",    110, 0,   255)
gen.add("max_err_rect", double_t, 0, "name",    110, 80,   200)
gen.add("max_err_ellipse", double_t, 0, "name",    110, 80,   200)
gen.add("k_err_rect", double_t, 0, "name",    0.3, 0.1,   0.5)
gen.add("k_err_ellipse", double_t, 0, "name",    0.3, 0.1,   0.5)
exit(gen.generate(PACKAGE, "image_converter", "Cam"))
